workflow/snakemake_rules/common.smk

```diff
@@ -164,18 +164,20 @@ def _get_upload_inputs(wildcards):
     # mapping of remote → local filenames
     uploads = {
         f"aligned.fasta.xz":              f"results/aligned_{origin}.fasta.xz",              # from `rule align`
-        f"sequence-diagnostics.tsv.xz":   f"results/sequence-diagnostics_{origin}.tsv.xz",   # from `rule diagnostic`
-        f"flagged-sequences.tsv.xz":      f"results/flagged-sequences_{origin}.tsv.xz",      # from `rule diagnostic`
-        f"to-exclude.txt.xz":             f"results/to-exclude_{origin}.txt.xz",             # from `rule diagnostic`
         f"masked.fasta.xz":               f"results/masked_{origin}.fasta.xz",               # from `rule mask`
         f"filtered.fasta.xz":             f"results/filtered_{origin}.fasta.xz",             # from `rule filter`
         f"mutation-summary.tsv.xz":       f"results/mutation_summary_{origin}.tsv.xz",       # from `rule mutation_summary`
     }
 
     for build_name in config["builds"]:
         uploads.update({
-            f"{build_name}/sequences.fasta.xz": f"results/{build_name}/{build_name}_subsampled_sequences.fasta.xz",
-            f"{build_name}/metadata.tsv.xz":    f"results/{build_name}/{build_name}_subsampled_metadata.tsv.xz",
+            f"{build_name}/sequences.fasta.xz": f"results/{build_name}/{build_name}_subsampled_sequences.fasta.xz",   # from `rule combine_samples`
+            f"{build_name}/metadata.tsv.xz":    f"results/{build_name}/{build_name}_subsampled_metadata.tsv.xz",      # from `rule combine_samples`
+            f"{build_name}/aligned.fasta":      f"results/{build_name}/aligned.fasta",                                # from `rule build_align`
```

Comment:
Should we make this `aligned.fasta.xz` to be consistent with everything else, incl. the top-level `aligned.fasta.xz`? Seems like it, to me.

Reply:
Typo. Fixed.

Reply:
Misread your original comment @tsibley - I've now resolved this via 69c4f77. I needed to add a new `compress_build_align` rule to generate the `results/{build_name}/aligned.fasta.xz` file which we then upload.